mu4e/mu4e-view.el

```diff
@@ -362,14 +362,17 @@ article-mode."
     (unless marked-read
       ;; when we're being marked as read, no need to start rendering the messages; just the minimal
       ;; so (update... ) can find us.
-      (mm-disable-multibyte)
       (insert-file-contents-literally path)
+      (message-narrow-to-headers-or-head)
+      (if (message-fetch-field "Content-Type" t)
+          (widen)
+        ;; For example, for messages in `mu4e-drafts-folder'
+        (erase-buffer)
+        (mm-enable-multibyte)
+        (insert-file-contents path))
       (setq
```

Comment:
Can you explain what this block does and esp., why?

Reply:
It inserts the message with multibytes turned on.  This enables the interpretation of UTF-8 byte sequences for draft messages — which do not have a charset.

Reply:
Thanks, I see... can you add a comment tin the code that explains that?

Do we really need to re-load the file from disk (`insert-file-contents`)? Would be nice if we could avoid that.

Reply:
Thanks for prompting me to do a better job! ☺  I reworked the commit not to read the file twice.